src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+  let baseDiscount = 0;
+
+  if (days <= 0) {
```

Comment:
I guess it's a good option, in this task you will not get invalid data, but I would throw error in that case